minecraft/proxy/java/viaaas/egg-v-i-aaa-s.json

```diff
@@ -0,0 +1,50 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-06-28T11:55:37-04:00",
+    "name": "VIAaaS",
+    "author": "regulad@outlook.com",
+    "description": "VIAaaS - ViaVersion as a Service - Standalone ViaVersion proxy",
+    "features": null,
+    "images": [
+        "ghcr.io\/pterodactyl\/yolks:java_16"
+    ],
+    "file_denylist": [],
+    "startup": "java -Xms128M -Xmx{{SERVER_MEMORY}}M -jar {{SERVER_JARFILE}} -sslPort {{WEBSERVER_PORT}}",
+    "config": {
+        "files": "{\r\n    \"config\/viaaas.yml\": {\r\n        \"parser\": \"yaml\",\r\n        \"find\": {\r\n            \"port\": \"{{server.build.default.port}}\",\r\n            \"bind-address\": \"0.0.0.0\"\r\n        }\r\n    }\r\n}",
+        "startup": "{\r\n    \"done\": \"Application started:\"\r\n}",
+        "logs": "{\r\n    \"custom\": false,\r\n    \"location\": \"logs\/latest.log\"\r\n}",
+        "stop": "end"
+    },
+    "scripts": {
+        "installation": {
+            "script": "#!\/bin\/bash\r\n# VIAaaS Installation Script\r\n#\r\n# Server Files: \/mnt\/server\r\napt update\r\napt install -y curl\r\n\r\ncd \/mnt\/server\r\n\r\ncurl -Lf -o ${SERVER_JARFILE} \"https:\/\/jitpack.io\/com\/github\/ViaVersion\/VIAaaS\/master-SNAPSHOT\/VIAaaS-master-SNAPSHOT-all.jar\"",
+            "container": "debian:buster-slim",
+            "entrypoint": "bash"
+        }
+    },
+    "variables": [
+        {
+            "name": "VIAaaS Jar File",
+            "description": "The name of the jarfile to be used when downloading & running VIAaaS.",
+            "env_variable": "SERVER_JARFILE",
+            "default_value": "VIAaaS-all.jar",
+            "user_viewable": true,
+            "user_editable": true,
```

Comment:
Is there a reason, why you've allow the user to change the binary name?

Reply:
Most other Minecraft eggs allow changing the binary. If the user wants to update it themselves, for example.